book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(author, title)
+    @author = author
+    @title = title
+    @rentals = [] # Initialize an empty array to store rentals
+  end
```

Comment:
- [ ] nice job. Please ensure that a method for adding rentals is present. Here, the book itself should be sent as a parameter to create the new rental, achieving the 'has-many' association
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```